docker/pytorch/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN curl -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest
      /opt/conda/bin/conda clean -ya
 ENV PATH /opt/conda/bin:$PATH
 # This must be done before pip so that requirements.txt is available
+RUN git clone https://github.com/pytorch/pytorch.git /opt/pytorch
```

Comment:
This actually isn't necessary since we do a `COPY . .` in the `/opt/pytorch` directory on Line 26

Reply:
But without this line docker build doesn't work, because after COPY .git file in /opt/pytorch doesn't exists. I think it's problem because this Dockerfile lies not in root of pytorch repo.

Also what difference between this dockerfile and dockerfile in root of pytorch repo?